pymfe/statistical.py

```diff
@@ -945,23 +1047,26 @@ def ft_nr_cor_attr(cls,
                        epsilon: float = 1.0e-8,
                        abs_corr_mat: t.Optional[np.ndarray] = None
                        ) -> t.Union[int, float]:
-        """Number of highly correlated pair distinct of attributes.
+        """Compute the number of highly correlated pair distinct of attributes.
```

Comment:
Maybe it's missing an 'of' between 'pair' and 'distinct', or maybe this could be refactored to 'Compute the number of distinct highly correlated pair of attributes.'

Reply:
fixed in 4f18360